openapi/connector_mgmt.yaml

```diff
@@ -1009,6 +1009,391 @@ paths:
                   $ref: "#/components/examples/500Example"
           description: An unexpected error occurred creating the connector namespace
 
+  /api/connector_mgmt/v1/processors:
```

Comment:
IMHO, procesors should be on a v2alpha1 (or something like that) API

Reply:
OK. Fair enough.

I'll expose the new agent endpoints on that context path too.

Presumably changes to the Fleet Shard to support use of different versioned API endpoints is to be added.

Presumably this will be by whoever works on the Fleet Shard changes for Processors.